config/locales/views/en.yml

```
@@ -734,8 +734,12 @@ en:
         random_in_collection: These are some random tags used in the collection.
       search_tags: try our tag search
     show:
+      canonical_html: It's a %{canonical_tag_faq_link}. You can use it to %{filter_works_link} and to %{filter_bookmarks_link}.
+      canonical_tag_faq: canonical tag
```

I would expect `canonical_tag_faq` to have the contents "canonical tag FAQ" or similar. IMO the following is a bit clearer as to the key's function
```suggestion
      canonical_tag: canonical tag
```